robot_smach_states/src/robot_smach_states/navigation/follow_operator2_0.py

```diff
@@ -433,19 +433,17 @@ def __init__(self, robot):
 
 
 if __name__ == "__main__":
+
+    from robot_skills import get_robot
+
     if len(sys.argv) > 1:
```

Comment:
Add argument check like other files to be consistent

Reply:
See a71172c306547eba6d50c057baaef8c0ec10a1d0 and 106535870505f785620744139c45820da4f330a4